sfc/application.txt

```diff
@@ -0,0 +1,632 @@
+      Employed by elego Software Solutions GmbH (2011 - 2013)
+
+    - Patrick Steinhardt <ps@pks.im>, active 2015 - present
+      Employed by elego Software Solutions GmbH (2015 - present)
+
+    - Ben Straub <bs@github.com>, active 2012 - 2014
+      Employed by GitHub Inc (2012 - 2014)
+
+    - Edward Thomson <ethomson@edwardthomson.com>, active 2012 - present
+      Employed by Microsoft Corp (2012 - 2015)
+      Employed by GitHub Inc (2016 - 2017)
+      Employed by Microsoft Corp (2017 - present)
+
+    Contributors to the LibGit2Sharp project (who are not included
+    above):
+
+    - Tim Clem <timothy.clem@gmail.com>, active 2011 - 2012
+      Employed by GitHub Inc (2011 - 2012)
+
+    - Keith Dahlby <dahlbyk@gmail.com>, active 2011 - present
+      Employed by J&P Cycles (2011 - 2016)
+      Employed by HuBoard (2016 - present)
+
+    - Yoram Harmelin <yoram.harmelin@gmail.com>, active 2012 - 2014
+      Employed by AXA (2012 - 2014)
+
+    - Jameson Miller <jamill@microsoft.com>, active 2012 - 2015
+      Employed by Microsoft Corp (2012 - 2015)
+
+    - Marius Ungureanu <marius.ungureanu@xamarin.com>, active 2014 - present
+      Employed by Xamarin Inc (2014 - 2016)
+      Employed by Microsoft Corp (2016 - present)
+
+    Contributors to the NodeGit project (who are not included above):
+
+    - Josh Abernathy <joshaber@gmail.com>, active 2015 - present
+      Employed by GitHub Inc (2015 - present)
+
+    - Tim Branyen <tim@tabdeveloper.com>, active 2011 - present
+      Employed by Netflix (2015 - present)
+      Employed by Bocoup (2011 - 2015)
+
+    - John Haley <johnh@axosoft.com>, active 2014 - present
+      Employed by Axosoft (2014 - present)
+
+    - Nick Kallen <socialmediamaster9000@gmail.com>, active 2013
+      Employed by Twitter (2013)
+
+    - Stjepan Rajko <stjepanr@axosoft.com>, active 2015 - present
+      Employed by Axosoft (2015 - present)
+
+    - Michael Robinson <mike@pagesofinterest.net>, active 2013
+      Employed by PAN Media and Advertising (2013)
+
+    - Max Korp <maxkorp@8bytealchemy.com>, active 2014 - present
+      Employed by Axosoft (2014 - present)
+
+    - Tyler Wanek <tylerw@axosoft.com>, active 2015 - present
+      Employed by Axosoft (2015 - present)
+
+    - Steve Smith <steve@github.com>, active 2014 - 2015
+      Employed by GitHub Inc (2014 - 2015)
+
+    Contributors to the ObjectiveGit project (who are not included above):
+
+    - Piet Brauer <piet@nerdishbynature.com>, active 2015 - present
+      Employed by Nerdish by Nature (2015 - present)
+
+    - Ben Chatelain <ben@octop.ad>, active 2014 - present
+      Employed by Kaiser Permanente (2014 - present)
+
+    - Slava Karpenko <slavikus@gmail.com>, active 2015 - present
+      Employed by Unreal Mojo (2015 - present)
+
+    - Alan Rogers <alanjrogers@me.com>, active 2014
+      Employed by GitHub (2014)
+
+    - Etienne Samson <samson.etienne@gmail.com>, active 2013 - present
+      Employed by Mediapart (2013 - present)
```

Comment:
Heh, that's an homonym, I've never worked at Mediapart myself 🤣. `Employed by Kokopelli (2012 - 2019).`

Reply:
Thanks!  Sorry that we erred here.